main.tf

```
@@ -0,0 +1,19 @@
+resource "aws_account_alternate_contact" "billing" {
+  
+  alternate_contact_type = "billing"
+  count = var.aws_account_alternate_contacts.billing != null ? 1 : 0
+}
+
+resource "aws_account_alternate_contact" "operations" {
+  
+  alternate_contact_type = "operations"
+  count = var.aws_account_alternate_contacts.operations != null ? 1 : 0
+
+}
+
+resource "aws_account_alternate_contact" "security" {
+
+  alternate_contact_type = "security"
+  count = var.aws_account_alternate_contacts.security != null ? 1 : 0
+
+}
```

Git sometimes has this weird thing about files without a newline terminating the file. So, GitHub flags it.

```suggestion
}

```

---

updated